src/background/popupHandler.ts

```diff
@@ -8,6 +8,7 @@ import {
   POPUP_TYPE_RAW_SIGN,
 } from '../popup/utils';
 
+// @ts-ignore
```

Comment:
I don't like this solution. I'm open for the suggestions.

Reply:
What kind of problem it tries to solve?

Reply:
https://github.com/aeternity/superhero-wallet/actions/runs/5356614184/jobs/9716655449#step:9:104

Reply:
If this is caused by adding `strictFunctionTypes` to the TS config file maybe we should remove it to avoid this errors?

Reply:
[This is necessary to avoid type issues coming from sdk, and a part of sdk13 usage.](https://github.com/aeternity/aepp-sdk-js/blob/0c9386096701a238edcc55c09287b4487a865a95/docs/README.md?plain=1#L85)

Reply:
OK< didn't know that. Resolving.